gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaJobStatusMonitor.java

```diff
@@ -230,7 +235,16 @@ protected void processMessage(DecodeableKafkaRecord<byte[],byte[]> message) {
           if (this.dagProcEngineEnabled && DagProcUtils.isJobLevelStatus(jobName)) {
             if (updatedJobStatus.getRight() == NewState.FINISHED) {
               // todo - retried/resumed jobs *may* not be handled here, we may want to create their dag action elsewhere
-              this.dagManagementStateStore.addJobDagAction(flowGroup, flowName, flowExecutionId, jobName, DagActionStore.DagActionType.REEVALUATE);
+              try {
+                this.dagManagementStateStore.addJobDagAction(flowGroup, flowName, flowExecutionId, jobName, DagActionStore.DagActionType.REEVALUATE);
+              } catch (Exception e) {
+                if (isExceptionInstanceOf(e, nonTransientExceptions)) {
+                  // todo - add metrics
+                  log.error(e.getMessage());
```

Comment:
`log.warn` is probably more appropriate.  also would be good to preface the exception msg w/ context lead-in (especially as there's no stack trace for figuring out where in the code did the logging!).  in addition, let's mention that the error is being swallowed/ignored.